platform/android/MapboxGLAndroidSDKWearTestApp/src/main/java/com/mapbox/weartestapp/utils/OffsettingHelper.java

```diff
@@ -34,7 +34,7 @@ public void updateChild(View child, WearableRecyclerView parent) {
   }
 
   @Override
-  protected void adjustAnchorOffsetXY(View child, float[] anchorOffsetXY) {
+  public void adjustAnchorOffsetXY(View child, float[] anchorOffsetXY) {
```

Comment:
to double check if the wear module was working correctly I ran it on an emulator but couldn't compile as AS showed this as a violation of visibility of the base class method. (I added that as second part of the commit message: `... + fixup wear compile error`